src/ascii.rs

```diff
@@ -154,12 +154,17 @@
 //! # }
 //! ```
 
+pub mod chain;
 pub(crate) mod checksum;
 mod command;
+pub mod data_type;
 mod id;
+mod marker;
 pub mod parse;
 mod port;
 mod response;
+pub mod scope;
+pub mod setting;
```

Comment:
I wonder if this is getting too busy. Consider how to make the API easier to navigate.

Reply:
I don't think we can solve this in this PR. I think it will be easier to clean up the module structure once I know more of what will be added in future PRs.